bindings/node/test/clientEncryption.test.js

```diff
@@ -32,7 +35,7 @@ describe('ClientEncryption', function() {
   });
 
   beforeEach(function() {
-    if (process.env.MONGODB_NODE_SKIP_LIVE_TESTS) {
+    if (requirements.SKIP_LIVE_TESTS) {
```

Comment:
looks like you might need to rebase

Reply:
This is an intentional change I made in 2c903d8ea45d2c1eb6af30b5929a480b14e28fae. `requirements.SKIP_LIVE_TESTS === process.env.MONGODB_NODE_SKIP_LIVE_TESTS`